### PR TITLE
Create search with lilo

### PR DESCRIPTION
**Pull request by sy-per**
Add lilo with browser please

### CODE REVIEW

**Comment by mateosalta** [comment]
hi, there is a "get the app" from android banner at the bottom, you would need to create a userscript to remove this from css

**Comment by sy-per** [comment]
Hello,
Since I use it on my phone (meizu pro 5 ubports last ota) I'm not "get the app" from android banner ...

File: src/app/webbrowser/searchengines/lilo.xml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+<OpenSearchDescription xmlns="http://a9.com/-/spec/opensearch/1.1/">
+  <ShortName>Lilo</ShortName>
+  <Description>Lilo. Search by Lilo.</Description>
```
Comment (Flohack74) [inline src/app/webbrowser/searchengines/lilo.xml]: Would it not be better "Search with Lilo" ?

**Review by Flohack74 — APPROVED** [review]

**Comment by mateosalta** [comment]
maybe you have desktop mode set?

**Comment by mateosalta** [comment]
screenshot20190316_124430366

**Comment by UniversalSuperBox** [comment]
@sy-per, is there any specific way you would like to be credited in our changelog for our next release? Would you prefer your username, the real name you have set on GitHub..?

**Pull request merged**